src/app/zap-templates/zcl/data-model/chip/wifi-network-management-cluster.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+    <domain name="CHIP"/>
+
+    <cluster>
+        <domain>Network Infrastructure</domain>
+        <name>Wi-Fi Network Management</name>
+        <code>0x0C00</code>
```

Comment:
This does not seem to match the spec.

Reply:
Ah good catch, I thought I had updated the ID in the xml but apparently I didn't.